articles/desktop-flows/virtual-desktops.md

```diff
@@ -14,117 +14,97 @@ search.audienceType:
   - enduser
 ---
 
-# Automating on virtual desktops (preview)
+# Automate on virtual desktops (preview)
 
 [!INCLUDE[cc-beta-prerelease-disclaimer](../includes/cc-beta-prerelease-disclaimer.md)]
 
-Apart from physical machines, desktop flows enable you to automate the operating system and applications in Citrix and RDP (Microsoft Remote Desktop Protocol) virtual desktops, and interact with Citrix virtual apps.
-
-Automating on virtual desktops works precisely as on physical machines. You can [capture UI elements](ui-elements.md), [deploy UI automation actions](actions-reference/uiautomation.md), and [create desktop flows using the recorder](recording-flow.md).
+With the Power Automate agent for virtual desktops, you can automate processes on virtual desktops just as easily as you can on physical devices. If your virtual desktop uses Citrix or Microsoft Remote Desktop Protocol (RDP), you can [capture UI elements](ui-elements.md), [deploy UI automation actions](actions-reference/uiautomation.md), and [create desktop flows using the recorder](recording-flow.md), just like on your physical desktop.
```

Comment:
@rhanajoy Here is really important to mention that users can automate also Citrix virtual apps. It's a different thing that Citrix and RDP.

Reply:
Ah! Agreed. Then I would change the first sentence to:
With the Power Automate agent for virtual desktops, you can automate processes on virtual desktops and interact with Citrix virtual apps just as easily as you can on physical devices.

Reply:
It sounds perfect, thank you!